tests/keras/backend/backend_test.py

```diff
@@ -36,6 +35,9 @@
     warnings.warn('Could not import the Theano backend')
 
 
+WITH_NP = [KTH if K.backend() == 'theano' else KC if K.backend() == 'cntk' else KTF, KNP]
```

Comment:
I must be missing something. Why this rather than `WITH_NP = [K, KNP]`?

Reply:
I see, it's to be able to compare it with the other backends with `==`. Nevermind.